src/DatabaseGpt.SqlServer/SqlServerDatabaseGptProvider.cs

```diff
@@ -38,7 +38,7 @@ public async Task<IDataReader> ExecuteQueryAsync(string query)
         }
         catch (SqlException ex)
         {
-            throw new DatabaseGptException("Something went wrong", ex);
+            throw new DatabaseGptException("An error occured. Se inner exception for details", ex);
```

Comment:
There is a typo. Rewrite the message as it sounds like the typical error messages of this kind, something like `An error occurred while executing the query. See the inner exception for details`.